src/herbivoresAndCarnivores.test.js

```diff
@@ -124,4 +121,36 @@ describe('Animal', () => {
     expect(Animal.alive.includes(tiger))
       .toBe(true);
   });
+
+  it(`Herbivores should be removed from 'Animal.alive' array, when their health is 0 (but Zebra is not the last defined animal)`, () => {
+    const zebra = new Herbivore('Zebra');
+    const tiger = new Carnivore('Tiger');
+
+    tiger.bite(zebra);
+    tiger.bite(zebra);
+
+    expect(Animal.alive.includes(zebra))
+      .toBe(false);
+
+    expect(Animal.alive.includes(tiger))
+      .toBe(true);
+  });
+
+  it('Two of the same herbivore exist, but only one dies', () => {
+    const olderZebra = new Herbivore('Zebra');
+    const yongerZebra =  new Herbivore('Zebra');
+    const lion = new Carnivore("Lion");
+
+    lion.bite(yongerZebra);
+    lion.bite(yongerZebra);
```

Comment:
Verify that the 'Animal.alive' array properly handles the case where multiple animals with the same name exist but only one dies.